tests/mlp_models.py

```diff
@@ -66,7 +66,7 @@ def dynamic_regressor(
     n_features_in_ = meta["n_features_in_"]
     n_outputs_ = meta["n_outputs_"]
 
-    compile_kwargs["loss"] = compile_kwargs["loss"] or KerasRegressor.r_squared
+    compile_kwargs["loss"] = compile_kwargs["loss"] or "mse"
```

Comment:
Yikes. This fix is needed because minimizing R^2 is the same as maximizing MSE.

Reply:
Yep! A huge upsie. It wasn't caught until now because (1) this is in tests, so no users were experiencing this and (2) none of the tests were actually checking scores.